modules/iothub/devdoc/iothub.md

```diff
@@ -22,6 +22,16 @@ The IotHub module dynamically creates per-device instances of IoTHubClient. It c
 IoTHubClient. Note that the AMQP and HTTP transports will share one TCP connection for all devices; the MQTT transport will create a new
 TCP connection for each device.
 
+#### Device registration
+
+In some circumstances, a device may not want to send telemetry but receive cloud to device messages.  WHen a device wants to connect to the IoTHub but but send a telemetry message, it may send a message with the following properties:
```

Comment:
Nit: Minor fixes:

> In some circumstances, a device may not want to send telemetry but receive cloud to device messages.  When a device wants to connect to the IoTHub and wishes to signal that it is expecting to receive cloud-to-device messages, it may send a message with the following properties: